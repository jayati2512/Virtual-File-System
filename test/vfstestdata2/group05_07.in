mountvfs vfs1
createvfs vfs1
mountvfs vfs1
unmountvfs vfs1
deletedir /another/abc/ss
createvfs vfs1 1024
mountvfs vfs1
deletedir /another/abc
makedir /another /abc
makedir /as /aa
movedir /another/abc /as/aaa
deletedir /another/abc/as
updatefile /another/abc/one.txt 
unmontvfs vfs2
unmountvfs vfs1
