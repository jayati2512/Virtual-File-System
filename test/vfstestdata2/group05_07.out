mountvfs_FAILURE DATA_FILE_NOT_FOUND
createvfs_FAILURE VFS_INSUFFICIENT_ARGUMENTS
mountvfs_FAILURE DATA_FILE_NOT_FOUND
unmountvfs_FAILURE DATA_FILE_NOT_FOUND
deletedir_FAILURE VFS_NOT_MOUNTED
createvfs_SUCCESS
mountvfs_SUCCESS
deletedir_FAILURE CANNOT_FIND_SPECIFIED_PATH_OR_DIR
makedir_SUCCESS
makedir_SUCCESS
movedir_SUCCESS
deletedir_FAILURE DIRECTORY_IS_NOT_EMPTY
updtefile_FAILURE VFS_INSUFFICIENT_ARGUMENTS
unmountvfs_FAILURE VFS_NOT_MOUNTED
unmountvfs_SUCCESS
